### ISSUE TITLE
Not Stretching to Fill Screen!

### ISSUE THREAD

**Issue by FRANKENSTEIN888** [Rosalie241/RMG#182]
I have tried Stretch to fill Screen and other aspect ratios and windows resolution and Your N64 Emulator It will not stretch to fit screen on SteamDeck.

**Comment by GAMECUBE-N64** [comment]
All There is a Small Picture with "FAT" black bars around it. Stretch to fit not working or other Aspect Ratios for filling Steamdeck Screen.

**Comment by Calinou** [comment]
This is likely a duplicate of https://github.com/Rosalie241/RMG/issues/123.

**Comment by FRANKENSTEIN888** [comment]
What the Hell are You talking about. Your not making any sense. Im glad I am going back to Paralel! At there Emulator works the way its supposed to and it uses Vulkan Engine unlike Yours!

**Comment by FRANKENSTEIN888** [comment]
What Do You Mean????? This just started for me after last update. Before that there was no problem with it full screen! You mentioning Duplicate #123 is long before I had problems with it not going Full Screen anymore!

**Comment by FRANKENSTEIN888** [comment]
With lack of Direction and help from the Creator! And dicking around several days with the settings. I found my ow fix in settings. Go to Graphic settings. Then Emulation. Set Render 2D Elements in N64 Emulation Enable Optimized and save. Then go into regular settings to Interface. Then Check Automatically Switch to Full Screen on Emulation Start. This Emulator is supposed to be a cousin to Mupen64 FZ but it is not. On Mupen64 FZ You set Stretch to fit screen on any Android Gaming Device it will Automatically do what it is supposed to. None of this "NONSENSE" that I had to go thru. Have a Nice Day!
